package.json

```diff
@@ -18,7 +18,8 @@
     "on-headers": "~1.0.1",
     "parseurl": "~1.3.1",
     "uid-safe": "~2.1.0",
-    "utils-merge": "1.0.0"
+    "utils-merge": "1.0.0",
+    "parse-domain": "^0.2.1"
```

Comment:
Please use an exact version and order this to be alphabetical (otherwise `npm` operations will reorganize it, creating unnecessary churn).